modules/electron/system-part1/src/cellular_hal.cpp

```diff
@@ -8,3 +8,4 @@
 #include "../src/electron/inet_hal_new.cpp"
 #include "../src/electron/socket_hal.cpp"
 #include "../shared/cellular_sig_perc_mapping.cpp"
+#include "../src/electron/platform_ncp_electron.cpp"
```

Comment:
I still dont fully understand the appropriate way to work with dynalib, particularly with Gen2 devices, but this is how I was able to get my builds to link successfully. If this is not the correct way to do things let me know

Reply:
:+1: Due to the weirdness in dependency order, this is the right hack.